src/components/DateTimeParseErrorModal.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+    <ion-header>
+    <ion-toolbar>
+      <ion-buttons slot="start">
+        <ion-button @click="closeModal"> 
+          <ion-icon :icon="close" />
+        </ion-button>
+      </ion-buttons>
+      <ion-title>{{ $t("Date time parse error") }}</ion-title>
+    </ion-toolbar>
+  </ion-header>
+
+  <ion-content class="ion-padding">
```

Comment:
Remove padding here and everywhere else.

Reply:
Removed.